vsintegration/src/FSharp.Editor/Common/SignatureGenerator.fs

```diff
@@ -0,0 +1,1055 @@
+﻿module Microsoft.VisualStudio.FSharp.Editor.SignatureGenerator
```

Comment:
This code is indeed a major duplication of the NicePrint.fs functionality.  I believe we need to have one and only one implementation of the process of signature printing and formatting, otherwise we are headed for major maintenance problems.

Please work out how to use the NicePrint.fs code instead of this code